robot_skills/src/robot_skills/util/entity.py

```diff
@@ -239,6 +239,10 @@ def from_entity_info(e):
         and e.id != "floor" and "wall" not in e.id:
         super_types += ["furniture"]
 
+    if all([v in volumes for v in ["handle", "frame_left_point", "frame_right_point"]]):
+        super_types += ["door"]
+        super_types.remove("furniture")
+
```

Comment:
This should move to `ed_python`

Reply:
I applied https://github.com/tue-robotics/tue_robocup/pull/1147, but this isn't applied yet.

We should first find a generic solution for entity inheritance classes.